tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -273,6 +273,40 @@ def test_combine_tokens_into_words(self):
         self.assertEqual(expected_tokens, output_rust[1])
         self.assertEqual(expected_indices, output_rust[2])
 
+    def test_basic_normalizer(self):
+        tokenizer = self.get_tokenizer()
+        rust_tokenizer = self.get_rust_tokenizer()
+
+        input_str = "Hola güey!"
+        expected_output_normalize = "hola güey "
```

Comment:
Note: this trailing space is an artefact of the punctuation that has been removed (this is consistent with the original Whisper implementation)